.travis.yml

```diff
@@ -9,6 +9,13 @@ rvm:
   - 2.3.0
   - ruby-head
   - rbx-2
+  - jruby-9.0.4.0
+  - jruby-9.0.5.0
```

Comment:
One `jruby-9xxx` should be enough IMO. About having `jruby-9xxx` and `jruby-head`, I think it's generally a good idea to keep both. On the other side, I wonder if keeping `2.0.0` is necessary.